src/procedures/TransferReservationOwnership.ts

```diff
@@ -18,11 +21,22 @@ export const createTransferReservationOwnershipResolver = (
   );
 };
 
+/**
+ * Procedure to transfer the ownership of a ticker reservation
+ */
 export class TransferReservationOwnership extends Procedure<
   TransferReservationOwnershipProcedureArgs
 > {
   public type = ProcedureType.TransferReservationOwnership;
 
+  /**
+   * - Transfer the ownership for a ticker reservation
```

Comment:
Hyphen not needed

Transfer ownership of a Security Token Reservation to the supplied address